frontend/angular.json

```diff
@@ -88,6 +89,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```

Comment:
Take a look at this:
![image](https://user-images.githubusercontent.com/66025811/85427782-3b943f00-b54a-11ea-992f-a78b4d35a9cc.png)

Reply:
The @import would be in `styles.css` so that you don't have to worry about removing or adding stylesheets in various places.

Reply:
Alright, gotchu